enzy_htp/structure/structure_io/xyz_io.py

```diff
@@ -28,15 +30,19 @@ def get_structure(self, path: str) -> Union[Structure, tuple]:
         pass
 
     @classmethod
-    def get_file_str(self, stru: Structure) -> str:
-        pass
-
+    @dispatch
+    def get_file_str(self, stru: Structure) -> List[str]:
```

Comment:
Please fix the type hinting of the return value

Reply:
You can use Union[Structure, StructureRegion] to make one dispatched function works for both input types. Please give it a try.